src/helpers/generate.ratings.js

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash';
+
+export default (ratingsPercentages, usersInfo, offset, limit) => {
+  // grouping all the users by the rating
+  const data = ratingsPercentages.data.map((percentageData) => {
+    const usersDetails = usersInfo[0]
+      .filter(userInfoData => userInfoData.rating === percentageData.rating);
+    return usersDetails;
+  });
+  //  mapping the array of ratings with all the information needed
```

Comment:
@nkpremices there's a lot going on here. Let's sync on Monday regarding this.

Reply:
Oh!! this file was supposed to be deleted now @MemunaHaruna. 
We don't need to sync. I've solved it with sequelize. 
I knew I deleted it. 
Really sorry. 
I'm gonna remove it

Reply:
I was using this file and that one of raw queries when I was using just one route. You actually told me that I should use two routes. and I did it